data_steward/validation/main.py

```diff
@@ -504,37 +510,33 @@ def process_hpo(hpo_id, force_run=False):
     """
     try:
         logging.info(f"Processing hpo_id {hpo_id}")
-        bucket = gcs_utils.get_hpo_bucket(hpo_id)
-        bucket_items = list_bucket(bucket)
-        folder_prefix = _get_submission_folder(bucket, bucket_items, force_run)
+        project_id = app_identity.get_application_id()
```

Comment:
It would be nice to see these outside the try block since they won't raise the exceptions caught.

Reply:
Nope, throws an exc